aten/src/ATen/native/cpu/DistanceOpsKernel.cpp

```diff
@@ -0,0 +1,192 @@
+#include "ATen/native/cpu/ReduceOpsKernel.h"
```

Comment:
If you are not using vectorized CPU instructions, please do not write your kernel in `native/cpu`.

Reply:
We should probably change this globally, maybe moving native/cpu to native/cpu/intrisic_ops or something.  It's confusing that CPU-only and code that works across CPU/CUDA unchanged both go in native.

Reply:
@cpuhrsch?

Reply:
@erikbrinkman Yes, please move it into native.

Reply:
@erikbrinkman - let's sync up on this offline. We had talked about whether you want to use vectorized instructions or not. Aside from this, I'll update the README within this folder and we'll probably want to move things or rename them just as greg suggested.

Reply:
We just talked about this in person and the conclusion is that it's worth benchmarking whether or not some level of auto-vectorization is beneficial here. If it's not, this can be moved back into native.